labels/labels.go

```diff
@@ -123,9 +123,9 @@ func (ls Labels) WithoutEmpty() Labels {
 // New returns a sorted Labels from the given labels.
 // The caller has to guarantee that all label names are unique.
 func New(ls ...Label) Labels {
-	set := make(Labels, 0, len(ls))
-	for _, l := range ls {
-		set = append(set, l)
+	set := make(Labels, len(ls))
+	for i, l := range ls {
+		set[i] = l
```

Comment:
That is strange I would think that the append would handle this the same way.

Can you maybe add a bench test and show the before and after this change?

Reply:
Yeah, I am pretty sure the go compiler optimizes this away in recent versions. Also, if optimizing this, I think we could get rid of the loop and just do:
```
copy(set, ls)
```